shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/openapi/OpenApiReaderImplementation.java

```diff
@@ -22,15 +22,17 @@ public static OpenApiReaderImplementation getInstance() {
      * Loads the stream of an OpenApi file and returns the output as a string.
      *
      * @param fileName Name of the file to load
+     * @param charset Charset used for the output string
      * @return File contents as string
      * @throws UnableToReadOpenApiSpecificationException If there is an issue loading the Api file
      */
     @Override
-    public String openAsString(String fileName) throws UnableToReadOpenApiSpecificationException {
-        InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName);
-        try {
-            return new String(
-                    Objects.requireNonNull(stream).readAllBytes(), StandardCharsets.UTF_8);
+    public String openAsString(String fileName, Charset charset)
+            throws UnableToReadOpenApiSpecificationException {
+        try (InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName)) {
+            var out = new String(Objects.requireNonNull(stream).readAllBytes(), charset);
+            stream.close();
```

Comment:
Is the `stream.close()` needed now that you're scoping the InputStream? Not sure about Java, but in C# the the resource would be closed when leaving the scope

Reply:
I don't believe the `stream.close()` is needed now.

Reply:
Updated - got rid of that closing call